Berlin/BerOr1270.xml

```diff
@@ -24,16 +30,265 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <msDesc xml:id="ms">
                <msIdentifier>
                   <repository ref="INS0333SBB"/>
-                  <collection>Ms. orient. oct.</collection>
+                  <collection>Ethiopic</collection>
                   <idno>SBP, Ms. orient. oct. 1270</idno>
                </msIdentifier>
                <msContents>
+                 <summary></summary>
 <msItem xml:id="ms_i1">
-  <locus from="2ra" to="24va"/>
-<title ref="LIT1450GadlaG">Gadla Gabra Krǝstos</title>
+  <locus from="2ra" to="16vb"/>
+  <locus from="19ra" to="24va"/>
+  <note>The folios of the manuscript are misplaced, being rearranged wrongly together: the text is in disorder.</note>
+  <title type="complete" ref="LIT1450GadlaG"/>
+  <textLang mainLang="gez"/>
+  <incipit xml:lang="gez">
+  <locus target="#2ra"></locus>
+  በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፨ ንቀድም፡ እንከ፡ በረድኤተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ንጽሕፍ፡ ዜናሁ፡ ለቅዱስ፡ ብእሴ፡ እግዚኣብሔር፡ <persName ref="PRS1661Alexis">ገብረ፡ ክርስቶስ</persName>።
+  ወልደ፡ <persName ref="PRS">ቴዎዶስዮስ፡</persName> ንጉሥ፡ ዘ<placeName ref="LOC2196Consta">ቈስጠንጥንያ</placeName>፨፨
+</incipit>
 </msItem>
-               </msContents>
-            </msDesc>
+<msItem xml:id="ms_i2">
+  <locus from="17ra" to="18vb"/>
+  <locus from="25ra" to="25vb"/>
+  <locus from="28ra" to="38va"/>
+  <note>Number of Miracles 8. The folios are misplaced, the text is in disorder. The right order of the miracles is: ff. 28ra-35vb, 36ra-36vb, 17ra-18vb, 37ra-37vb, 25ra-25vb, 38ra-38va. Folios missing between f. 35 and f. 36, with consequent missing part of the text.</note>
+  <title type="incomplete" ref="LIT2384Taamme"/>
+  <textLang mainLang="gez"/>
+<incipit xml:lang="gez">
+<locus target="#28ra"></locus>
+በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ንቀድም፡ እንከ፡ በረዴተ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፨ ወእምዝ፡ ንጽሕፍ፡ ተአምሪሃ፡ ለእግዝእትነ፡ ማርያም፡ ወላዲተ፡ አምላክ፡
+</incipit>
+<msItem xml:id="ms_i2.1">
+  <locus from="28rb" to="30ra"></locus>
+    <title type="complete"  subtype="miracle 1"></title>
+    <textLang mainLang="gez"></textLang>
+    <incipit xml:lang="gez">
+    <locus target="#28rb"></locus>
+    ወሀሎ፡ እንከ፡ አሐዱ፡ ብእሲ፡ በአሐቲ፡ ሀገር፡ ባዕል፡ ፈድፋድ፡ ወይገብር፡ ተዝካራ፡ ለማርያም፡ ወይቄርብ፡ ቍርባነ፡ በበ፡ በዓላ፡ ለማርያም፡
+    </incipit>
+    <note> The persoanl name <persName ref="PRS" xml:lang="gez">ኆላዊ፡ ክርስቶስ፡ <persName ref="PRS" xml:lang="gez" type="normalized">Ḫolāwi Krǝstos</persName></persName> is written (in red ink) in the supplication formula on <locus target="#30ra">, at the explicit of miracle 1.</locus></note>
```

Comment:
the markup with persName is here not completely according the BM guidelines. If you create a person ID (if you need help with creating, let us know), simply use `<persName ref="PRS1234dbfsklfj">Ḫolāwi Krǝstos</persName>`, no further mark up of languages is necessary. If you don't create an ID, no mark up at all is necessary.

Reply:
Yes, I don't understand how to create an ID. For the moment I don't mark them up, if they don't exist yet :-) (what an easy solution  from me!!!! :-),)

Reply:
I understand! So, I think you need to decide within your project which IDs you want to create. In BM, we create IDs for all persons and places mentioned in the manuscript (widely related to the manuscript production, so not necessarily the protagonists of texts or Biblical places etc.), but your project might chose another approach. If you decide not to mark them up, then simply use no persName at all.
The guidelines for creating new records are here: https://betamasaheft.eu/Guidelines/?id=newEntry. They are not complicated, but need to be read and followed closely for the first time. If you want to do it together on zoom,  let me know.

Reply:
Dorothea, I committed some corrections, but I need to work on the IDs miracles and incipits, and then I will pull request again, and maybe, if and when you can ,you could have a look again. Would it be ok?! Thanks a lot for all.

Reply:
Yes, that's perfect. If you want, to can keep this PR open and just write as a comment when you are finished with your corrections, or you can close the PR, continue working in the same branch and make a new PR when you're finished.

Reply:
Dorothea, I have completed to correct and revise. I did not pull request again (I could not do, I don't know). Could you still have a look to review it again?! Thank you.

Reply:
Thank you! But I cannot see any recent corrections. Are you sure you committed and pushed?